.github/workflows/featureRelease.yml

```diff
@@ -7,24 +7,48 @@ on:
     paths:
       - 'webapp_frontend/**'
       - 'webapp_provider/**'
+      - '.github/workflows/featureRelease.yml'
 
 jobs:
   Build_Docker_Image_on_Push:
     runs-on: ubuntu-latest
     steps:
       -
         name: Set up Project
-        uses: actions/checkout@v2
+        uses: actions/checkout@v1
+      -
+        name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11.0.8'
+          architecture: x64
```

Comment:
why java?

Reply:
Because the sonar scanner needs java and with the default version of the runner vm you cant access the keystore because of permissions issues. This installs Java to /opt where we have permissions to change stuff.